tanner/postgres_client.py

```diff
@@ -0,0 +1,94 @@
+import asyncio
+import logging
+
+import aiopg
+
+from tanner.config import TannerConfig
+
+LOGGER = logging.getLogger(__name__)
+
+
+class PostgresClient:
+    def __init__(self):
+        self.logger = logging.getLogger("tanner.db_helper.postgres")
+
+        self.host = TannerConfig.get("POSTGRES", "host")
+        self.port = TannerConfig.get("POSTGRES", "port")
+        self.user = TannerConfig.get("POSTGRES", "user")
+        self.password = TannerConfig.get("POSTGRES", "password")
+        self.db_name = TannerConfig.get("POSTGRES", "db_name")
+        self.poolsize = TannerConfig.get("POSTGRES", "poolsize")
+        self.timeout = TannerConfig.get("POSTGRES", "timeout")
+
+    async def get_pg_client(self):
+        pg_client = None
+        try:
+            dsn = "dbname={} user={} password={} host={} port={}".format(
+                self.db_name, self.user, self.password, self.host, self.port
+            )
+
+            pg_client = await asyncio.wait_for(
+                aiopg.create_pool(dsn, maxsize=self.poolsize), timeout=int(self.timeout)
+            )
+
+            async with pg_client.acquire() as conn:
```

Comment:
For me looks like mixing of connection creation and table routine

Reply:
This was the way recommended in the aiopg docs. So should I make a new function called `create_tables` and pass the `pg_client` object to that?